exotica/include/exotica/Tools.h

```diff
@@ -81,6 +82,12 @@ namespace exotica
   }
 
   std::string getTypeName(const std::type_info& type);
+
+  std::string parsePath(const std::string& path);
+
+  std::string loadFile(const std::string& path);
+
+  bool exists(const std::string& path);
```

Comment:
``exists`` is not necessarily clear - perhaps ``fileExists`` or ``folderExists`` or does it check either?